python/ray/_raylet.pyx

```diff
@@ -2057,7 +2086,7 @@ cdef CRayStatus task_execution_handler(
         except SystemExit as e:
             # Tell the core worker to exit as soon as the result objects
             # are processed.
-            if hasattr(e, "is_ray_terminate"):
+            if hasattr(e, "is_ray_terminate") and e.is_ray_terminate:
```

Comment:
Now we can have is_ray_terminate although is_ray_terminate = False so that we can add additional message when is_ray_terminate = False